inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/user/TenantRoleServiceImpl.java

```diff
@@ -116,4 +116,9 @@ public TenantRoleInfo getByUsernameAndTenant(String name, String tenant) {
         return CommonBeanUtils.copyProperties(entity, TenantRoleInfo::new);
     }
 
+    @Override
+    public boolean delete(Integer id) {
+        return tenantUserRoleEntityMapper.deleteById(id) > 0;
```

Comment:
No need to check if the delete result bigger than 0, just return it directly.

Reply:
done